packages/mobile/src/screens/track-screen/TrackScreen.tsx

```diff
@@ -146,13 +151,74 @@ export const TrackScreen = () => {
     return null
   }
 
+  const handlePressGoToRemixes = () => {
+    if (!remixParentTrack) {
+      return
+    }
+    navigation.push({
+      native: {
+        screen: 'TrackRemixes',
+        params: { id: remixParentTrack.track_id }
+      },
+      web: { route: trackRemixesPage(remixParentTrack.permalink) }
+    })
+  }
+
+  const remixParentTrackId = track.remix_of?.tracks?.[0]?.parent_track_id
+  const showMoreByArtistTitle =
+    (remixParentTrackId && lineup.entries.length > 2) ||
+    (!remixParentTrackId && lineup.entries.length > 1)
+
+  const hasValidRemixParent =
+    !!remixParentTrackId &&
+    !!remixParentTrack &&
+    remixParentTrack.is_delete === false &&
+    !remixParentTrack.user?.is_deactivated
+
+  const moreByArtistTitle = showMoreByArtistTitle && (
+    <Text
```

Comment:
lint issue i believe?

Reply:
I don't think we have this in the eslint config yet but I updated this to a ternary

Reply:
oh word, i was meaning less ternery but more just the jsx seemed funky, no worries tho